### PR TITLE
LOG-5709: Enable AzureMonitor test

### PR DESCRIPTION
**Pull request by Clee2691**
### Description
This test fixes deploying the functional framework and the Azure monitor output test.

/cc @cahartma @vparfonov 
/assign @jcantrill 

### Links
- JIRA: https://issues.redhat.com/browse/LOG-5709

### CODE REVIEW

File: test/functional/outputs/azuremonitor/forward_to_azuremonitor_test.go [hunk 1]
```diff
@@ -60,7 +60,6 @@ var _ = Describe("Forwarding to Azure Monitor Log ", func() {
 		nanoTime, _ := time.Parse(time.RFC3339Nano, timestamp)
 		message := "This is my new test message"
 		var appLogTemplate = functional.NewApplicationLogTemplate()
-		appLogTemplate.LogSource = logging.InfrastructureSourceContainer
```
Comment (jcantrill) [inline test/functional/outputs/azuremonitor/forward_to_azuremonitor_test.go:63]: Are we not pushing logsource in the datamodel?

**Review by jcantrill — COMMENTED** [review]
/approve

File: test/functional/outputs/azuremonitor/forward_to_azuremonitor_test.go (hunk 1 above)
Comment (Clee2691) [reply test/functional/outputs/azuremonitor/forward_to_azuremonitor_test.go:63]: `LogSource` was removed in [PR 2525](https://github.com/openshift/cluster-logging-operator/pull/2525/files#diff-87df0e8c62be9fc2a545c2cf34f3511c1fd14e7520f7fbd59bfdbd08f466534d):

**Pull request merged**